src/app/signup/signup.component.html

```diff
@@ -10,7 +10,7 @@ <h2>Participe hoje do Tuiter</h2>
                 [formControl]="signUpForm.controls['name']">
         </div>
 
-        <div class="form-group">
+        <div class="form-group" style="color:silver;">
```

Comment:
this is already included in the scss!

Reply:
Still doesn't affect a radio button unless it's set in the wrapping tag in the html for some reason.